packages/cli/src/zosfiles/compare/lfsdd/LocalfileSpooldd.definition.ts

```diff
@@ -0,0 +1,62 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+import { CompareOptions } from "../Compare.options";
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).COMPARE;
+
+/**
+ * Compare local-file-spool-dd command definition containing its description, examples and/or options
+ * @type {ICommandDefinition}
+ */
+export const LocalfileSpoolddDefinition: ICommandDefinition = {
+    name: "local-file-spool-dd",
+    aliases: ["lfsdd"],
```

Comment:
Should the alias be `lf-sdd` with a hyphen to be consistent with other new commands that have been added? 🙂